python/moz/l10n/formats/po/serialize.py

```diff
@@ -24,35 +25,41 @@
 def po_serialize(
     resource: Resource[str] | Resource[Message],
     trim_comments: bool = False,
-    wrapwidth: int = 78,
+    wrapwidth: int = 200,
```

Comment:
I assume we use 200 here because we use it at Mozilla broadly?

Non-blocking: It would be nice if this were configurable.